src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    class ScannerResultCustomListContext
+    {
+        internal Action UpdateTree { get; }
+        internal Action SwitchToServerLogin { get; }
+        internal bool ShowAllResults;
```

Comment:
Does `ShowAllResults` get used in the child, other than in `ChangeVIsibility` ?

Reply:
No, it is only used in `changeVisibility`.
I created this method to wrap `BtnShowAll` because it felt a little bit off exposing this UI element to the child, but I ended up wrapping `ShowAllResults` as well since it was just used to set the visibility.

Reply:
But seeing your next comment, I agree to leave this method on the parent side